tests/Doctrine/Tests/DBAL/Driver/Mysqli/MysqliConnectionTest.php

```diff
@@ -28,11 +28,33 @@ protected function setUp()
             ->getMockForAbstractClass();
     }
 
+    protected function tearDown()
+    {
+        if ($this->getName() == "testRestoresErrorHandlerOnException") {
```

Comment:
The issue here is that you have to do `restore_error_handler()` twice in the `testRestoresErrorHandlerOnException` test :-)